[MRG+1] Add and print a total_n_estimators information

Naereen opened the pull request:
Add to bagging.Bagging._parallel_build_estimators a total_n_estimators parameter, to improve the (optional) information printed if verbose > 1.

It felt weird to see 6 times in a row "building estimator 1 of 1" without any idea of total number of estimators to build, now it will be printed more nicely:
![typo__building_estimator_1_out_of_1](https://cloud.githubusercontent.com/assets/11994719/13762485/9559060e-ea3f-11e5-8ffd-0016f5805609.png)

Naereen committed:
Add and print a total_n_estimators information

Add to bagging.Bagging._parallel_build_estimators a total_n_estimators parameter, to improve the (optional) information printed if verbose > 1.

It felt weird to see 6 times in a row "building estimator 1 of 1" without any idea of total number of estimators to build, now it's fixed.

agramfort commented:
travis is not happy.

And I am +0.1 on this. I admit it's a private function hence the >0 + :)

GaelVaroquaux commented:
Indeed, the tests need to be fixed. But I am +1 for the feature

Naereen committed:
Two typo on my change, sorry

- total_n_estimators should be sum(n_estimators) not len(..)
- keyword argument should be after non-keyword argument (call of ``_parallel_build_estimators`` with ``delayed`` on line 365-366.

Naereen commented:
I saw the typo responsible for travis to yell at me -- sorry.
It sould be fixed now.

GaelVaroquaux commented:
LGTM. +1 for merge

agramfort commented:
Thx
